capstone-ui/src/components/GroupInbox/GroupInbox.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import GroupHeader from "../GroupHeader/GroupHeader";
+import "./GroupInbox.css";
+import axios from "axios";
+import { catchErrors } from "../../utils";
+
+export default function GroupInbox({ username }) {
+  const [inbox, setInbox] = React.useState(null);
```

Comment:
perhaps instead of starting as null, it could be an empty array? that way you don't need to check that it's not null when mapping the inbox.